app/views/admin/deploy_groups/show.html.erb

```diff
@@ -40,7 +40,16 @@
                     class: "btn btn-default"
         %>
         <%= link_to "Deploy All Projects", deploy_all_admin_deploy_group_path(@deploy_group), class: "btn btn-default", data: { method: "post" } %>
-      <% end %>
+        <%= link_to "Merge #{@deploy_group.cloned_stages.count} Cloned Stages",
+                    merge_all_stages_admin_deploy_group_path(@deploy_group),
+                    class: "btn btn-default",
+                    data: {
+                      method: "post",
+                      confirm: "Stages will be deleted. Before deleting, each stage will have its Deploy Group added to the stage it was copied from. Stages affected are those cloned from a template stage, have exactly this deploy group, no other deploy groups, and are on a project that uses include_new_deploy_groups.\n\n" +
```

Comment:
'have exactly this deploy group, no other deploy groups,' is a bit redundant ...